spec/bundler_inject_spec.rb

```diff
@@ -117,8 +117,8 @@
           expect(err).to match %r{^\*\* override_gem\("ansi", :git=>"https://github.com/rubyworks/ansi"\) at .+/bundler\.d/local_overrides\.rb:1$}
         end
 
-        it "with a path" do
-          with_path_based_gem("https://github.com/rubyworks/ansi") do |path|
+        it "with an absolute path" do
+          with_path_based_gem("https://github.com/rubyworks/ansi", "the_gem") do |path|
```

Comment:
More out of curiosity, why was this hardcoded to the parameter default?

Reply:
I had it as the default in the helper method because I wanted the interface to be backwards compatible.
In this spec, it referenced "the_gem", so I felt it was good to have this string in the spec.

We could probably drop the default.